### PR TITLE
samples client/server: log more consistently

### PR DESCRIPTION
**Pull request by jlevon**

### CODE REVIEW

File: samples/server.c [hunk 1]
```diff
@@ -68,7 +68,7 @@ struct server_data {
 static void
 _log(vfu_ctx_t *vfu_ctx UNUSED, UNUSED int level, char const *msg)
 {
-    fprintf(stderr, "server: %s\n", msg);
+    fprintf(stderr, "server%d: %s\n", getpid(), msg);
```
Comment (tmakatos) [inline samples/server.c:71]: nit: `server[%d]` ?

File: samples/server.c [hunk 2]
```diff
@@ -227,15 +227,15 @@ static void do_dma_io(vfu_ctx_t *vfu_ctx, struct server_data *server_data)
 
     memset(buf, 'A', count);
     get_md5sum(buf, count, md5sum1);
-    printf("%s: WRITE addr %p count %d\n", __func__,
+    vfu_log(vfu_ctx, LOG_DEBUG, "%s: WRITE addr %p count %d", __func__,
```
Comment (tmakatos) [inline samples/server.c:230]: This isn't called by the library so I think `printf` is fine.

File: samples/server.c [hunk 3]
```diff
@@ -227,15 +227,15 @@ static void do_dma_io(vfu_ctx_t *vfu_ctx, struct server_data *server_data)
...
            server_data->regions[0].iova.iov_base, count);
     ret = vfu_dma_write(vfu_ctx, &sg, buf);
     if (ret < 0) {
         errx(EXIT_FAILURE, "vfu_dma_write failed: %s\n", strerror(-ret));
     }
 
     memset(buf, 0, count);
-    printf("%s: READ  addr %p count %d\n", __func__,
+    vfu_log(vfu_ctx, LOG_DEBUG, "%s: READ  addr %p count %d", __func__,
```
Comment (tmakatos) [inline samples/server.c:238]: Ditto.

**Review by tmakatos — APPROVED** [review]
LGTM, a couple of minor nits.

File: samples/server.c (hunk 1 above)
Comment (jlevon) [reply samples/server.c:71]: better, thanks.

File: samples/server.c (hunk 2 above)
Comment (jlevon) [reply samples/server.c:230]: it's so we get the consistent prefix I added.

**Review by tmakatos — APPROVED** [review]

**Pull request merged**